tests/Unit/PointwiseFunctions/AnalyticSolutions/EinsteinSolutions/VerifyEinsteinSolution.hpp

```diff
@@ -158,11 +158,11 @@ void verify_time_independent_einstein_solution(
   // Compute derived spacetime quantities
   const auto upper_psi =
       gr::inverse_spacetime_metric(lapse, shift, upper_spatial_metric);
-  const auto christoffel_first_kind = compute_christoffel_first_kind(d4_psi);
+  const auto christoffel_first_kind_psi = gr::christoffel_first_kind(d4_psi);
```

Comment:
remove _psi

Reply:
That should work! I added _psi before christoffel_first_kind (removing compute_) was in a namespace. now the names should no longer conflict.

Reply:
Fixed.